assembly/plaintime.ts

```diff
@@ -64,4 +242,86 @@ export class PlainTime {
 
     return ord(a.nanosecond, b.nanosecond);
   }
+
+  add<T>(durationToAdd: T): PlainTime {
```

Comment:
```suggestion
  add<T = DurationLike>(durationToAdd: T): PlainTime {
```

Reply:
And after this you could remove all type hints in tests like `.add<DurationLike>({ ... })` to `.add({ ... })`

Reply:
Done